### DISCUSSION TITLE
Can we officially support vitest

### DISCUSSION THREAD

**Discussion by rverma-dev** [aws/aws-pdk#811] [General]
Trying to maintain cloudflare workers in same mono repo as of pdk, however the lack of support for vitest seems a blocker. 
Also I would like to avoid using both jest (cdk) and vitest (cloudflare). 
Request to consider opting for vitest/jest as part of monorepo/infrastructureProject setup

**Comment by agdimech** [comment]
Hi - i've actually been looking into Vite/Vitest over the last week or so and am eager to transition away from CRA in favour of Vite and adding support for Vitest as an alternative test framework. Not sure when this will be completed though.

Are you able to please raise a feature request so we can track this?

In the meantime, you could implement support for Vitest yourself by disabling jest and configuring the correct devDependencies and configuration if needed.
